plugins/WaveShaper/WaveShaper.cpp

```diff
@@ -82,7 +85,9 @@ bool WaveShaperEffect::processAudioBuffer( sampleFrame * _buf,
 	const float w = wetLevel();
 	float input = m_wsControls.m_inputModel.value();
 	float output = m_wsControls.m_outputModel.value();
-	const float * samples = m_wsControls.m_wavegraphModel.samples();
+	// getting the current graph samples (size: 200)
+	// getting the graph samples this often can cause lagg with bigger sample count (if automated or effected)
+	std::vector<float> graphSamples = m_wsControls.getGraphSamples();
```

Comment:
I will rework that. (and It is not because of that. It is because if the graph is automated then on every update a large amount of samples have to be recalculated)

Reply:
Updated this.